doc/whats_new.rst

```diff
@@ -27,15 +27,22 @@ Authors
 * `Stefan Appelhoff`_
 * `Adam Li`_
 * `Richard Höchenberger`_
+* `Austin Hurst`_
 
 Detailed list of changes
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
 Enhancements
 ^^^^^^^^^^^^
+<<<<<<< HEAD
```

Comment:
@adam2392 careful next time :)

Reply:
Oh no! 

oops... Is this fixable with just a push into master by one of the devs?

Reply:
one more note @adam2392 we now refer to PRs using the newly added sphinx substitution (#627):

```
(`#610 <https://github.com/mne-tools/mne-bids/pull/610>`_)
```

becomes
```
(:gh:`610`)
```